turbopy/core.py

```diff
@@ -540,6 +550,14 @@ def is_running(self):
         """Check if time is less than end time"""
         return self.this_step < self.num_steps
 
+    def __repr__(self):
+        clock_data = {
+            "start_time" : self.start_time,
+            "end_time" : self.end_time,
+            "num_steps" : self.num_steps,
+            "print_time" : self.print_time}
+        representation = f"{self.__class__.__name__}({clock_data})"
```

Comment:
You should just change `__init__` to save `clock_data` as `self.clock_data`. Then you can use `self.clock_data` here. This way, if future changes to this class introduce new parameters, this `__repr__` will still work.